osbs/cli/capture.py

```diff
@@ -20,18 +20,26 @@ class IterLinesSaver(object):
     Wrap HttpStream.iter_lines() and save responses.
     """
 
-    def __init__(self, path, fn):
+    def __init__(self, encoding, path, fn):
+        self.encoding = encoding or 'utf-8'
         self.path = path
         self.fn = fn
         self.line = 0
 
     def iter_lines(self):
+        encoding = self.encoding
         for line in self.fn():
             path = "{f}-{n:0>3}.json".format(f=self.path, n=self.line)
             logger.debug("capturing to %s", path)
+            try:
+                text = line.decode(encoding, errors='replace')
```

Comment:
Correct. `line` comes from `iter_lines()`, which returns encoded bytes ('bytes' in Python 3, 'str' in Python 2) -- note we're not passing `decode_unicode=True` here.